pkgs/build-support/cc-wrapper/ld-wrapper.sh

```diff
@@ -49,18 +54,17 @@ if [ "$NIX_ENFORCE_PURITY" = 1 -a -n "$NIX_STORE" \
     params=("${rest[@]}")
 fi
 
-LD=@prog@
```

Comment:
This however would be the wrapped `ld`. Since cc will fork ld-wrapper, the old `LD=...`, assuming `LD` was already an export so it affected forked processes' environments (!), would be clobbered anyways.